Makefile

```diff
@@ -0,0 +1,83 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DO NOT MANUALLY EDIT THIS FILE
+# =============
+
+# This script requires https://github.com/mikefarah/yq (not to be confused with https://github.com/kislyuk/yq)
+# It is available from Nixpkgs as `yq-go` (`nix shell nixpkgs#yq-go`)
+# This script also requires `jq` https://stedolan.github.io/jq/
+
+.PHONY: build publish
+
+TAG    := $(shell git rev-parse --short HEAD)
+OPERATOR_NAME := stackable-cockpit
+VERSION := $(shell cargo metadata --format-version 1 | jq -r '.packages[] | select(.name=="stackabled") | .version')
+
+DOCKER_REPO := docker.stackable.tech
+ORGANIZATION := stackable
+# this will be overwritten by an environmental variable if called from the github action
+HELM_REPO := https://repo.stackable.tech/repository/helm-dev
+HELM_CHART_ARTIFACT := target/helm/${OPERATOR_NAME}-${VERSION}.tgz
+
+SHELL=/usr/bin/env bash -euo pipefail
+
+render-readme:
+	scripts/render_readme.sh
```

Comment:
I don't think we need this. Also this will fail because the script `render_readme.sh` is not part of this PR.

Reply:
Also pulled in from templating but not currently invoked anywhere, and I assume we'll want to start using it _eventually_

Reply:
Okay, so we have two options:

- Remove above lines from the Makefile so that we cannot invoke the failing command
- Or we add the missing script to this PR to make it run-able

Just leaving it as is seems a little weird.

Reply:
I'll comment it out

Reply:
010950b2441b682e1d6857a7c29ce8e50bdd9e3a